doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -179,7 +179,7 @@ Gazell
 Matter
 ------
 
-|no_changes_yet_note|
+* Switched to using :kconfig:option:`CONFIG_PICOLIBC` as the C library instead of :kconfig:option:`CONFIG_NEWLIB_LIBC`, in compliance with Zephyr requirements.
```

Comment:
```suggestion
* Updated to using the :kconfig:option:`CONFIG_PICOLIBC` Kconfig option as the C library instead of :kconfig:option:`CONFIG_NEWLIB_LIBC`, in compliance with Zephyr requirements.
```